docs/run-random-beacon.adoc

```diff
@@ -0,0 +1,402 @@
+# Keep operator Ethereum account.
+[ethereum.account]
+  Address = "0xAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8AAAAAAAAA"
+  KeyFile = "/Users/someuser/ethereum/data/keystore/UTC--2018-03-11T01-37-33.202765887Z--AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA8AAAAAAAAA"
+
+# Keep contract addresses configuration.
+[ethereum.ContractAddresses]
+  # Hex-encoded address of KeepRandomBeaconOperator contract
+  KeepRandomBeaconOperator = "0xBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBBB"
+  # Hex-encoded address of TokenStaking contract
+  TokenStaking = "0xCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCCC"
+  # Hex-encoded address of KeepRandomBeaconService contract. Only needed
+  # in cases where the client's utility functions will be used (e.g., the
+  # relay subcommand).
+  KeepRandomBeaconService = "0xDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDDD"
+
+# Keep network configuration.
+[LibP2P]
+  Peers = ["/ip4/127.0.0.1/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1", "/dns4/some-keep-host.com/tcp/3919/ipfs/njOXcNpVTweO3fmX72OTgDX9lfb1AYiiq4BN6Da1tFy9nT3sRT2h1"]
+  Port  = 3920
+  # Override the node's default addresses announced in the network
+  AnnouncedAddresses = ["/dns4/example.com/tcp/3919", "/ip4/80.70.60.50/tcp/3919"]
+
+# Storage is encrypted
+[Storage]
+  DataDir = "/my/secure/location"
+```
+
+==== Parameters
+
+[%header,cols=4*]
+|===
+|`ethereum`
+|Description
+|Default
+|Required
+
+|`URL`
+|The Ethereum host your keep-client will connect to.  Websocket protocol/port.
+|""
+|Yes
+
+|`URLRPC`
+|The Ethereum host your keep-client will connect to.  RPC protocol/port.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.account`
+|Description
+|Default
+|Required
+
+|`Address`
+|The Keep operator Ethereum account address.
+|""
+|Yes
+
+|`KeyFile`
+|The local filesystem path to your Keep operator Ethereum account keyfile.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`ethereum.ContractAddresses`
+|Description
+|Default
+|Required
+
+|`KeepRandomBeaconOperator`
+|Hex-encoded address of the KeepRandomBeaconOperator Contract.
+|""
+|Yes
+
+|`KeepRandomBeaconService`
+|Hex-encoded address of the KeepRandomBeaconService Contract.
+|""
+|Yes
+
+|`TokenStaking`
+|Hex-encoded address of the TokenStaking Contract.
+|""
+|Yes
+|===
+
+[%header,cols=4*]
+|===
+|`LibP2P`
+|Description
+|Default
+|Required
+
+|`Peers`
+|Comma separated list of network peers to boostrap against.
+|[""]
+|Yes
+
+|`Port`
+|The port to run your instance of Keep on.
+|3919
+|Yes
+
+|`AnnouncedAddresses`
+|Multiaddr formatted hostnames or addresses annouced to the Keep Network. More on multiaddr format https://docs.libp2p.io/reference/glossary/#multiaddr[here].
+|[""]
+|No
+|===
+
+[%header,cols=4*]
+|===
+|`Storage`
+|Description
+|Default
+|Required
+
+|`DataDir`
+|Location to store the Keep nodes group membership details.
+|""
+|Yes
+|===
+
+== Build from Source
+
+See the https://github.com/keep-network/keep-core/tree/master/docs/development#building[building] section in our developer docs.
```

Comment:
```suggestion
See the <<docs/development/index.adoc#building,building>> section in
our developer docs.
```

This will resolve properly in GitHub across branches and on docs.keep.network.